.rspec_status

```diff
@@ -0,0 +1,78 @@
+example_id                          | status | run_time        |
```

Comment:
Should this file be in your .gitignore?

Reply:
yes, I think I must have overwritten it with the merge. Fixed now

Reply:
You should still probably remove this file from the commit.